### DISCUSSION TITLE
How can i export mutiple table with specified columns?

### DISCUSSION THREAD

**Discussion by bigwetguo** [mydumper/mydumper#1379] [Q&A]
my shell like this:
mydumper -u user -p pwd -h 192.168.12.30 -P 24601 -less-locking --no-locks --load-data -B gzzdb -T t_user --assign-columns user_id,user_name,user_add --fields-terminated-by '|!' --lines-terminated-by '\r\n', t_student --assign-columns student_id,student_name,student_class --fields-terminated-by '|#' --lines-terminated-by '\n' -o DATA_OUT/ -m

then show Error dumping table (t_user) data ：Unknow column 'student_id' in 'field list'

how can i export mutiple table with specified column?

**Comment by davidducos** [comment]
Hi @bigwetguo, I don't know what `--assign-columns ` is... 

You will be able to find on per table section https://github.com/mydumper/mydumper/blob/master/README.md#defaults-file what I think you are looking for, or you can check this: https://github.com/mydumper/mydumper/pull/1158

**Reply by bigwetguo** [reply]
LOL，maybe I download other moded version of your mydumper. --assign-columns can export designate columns data,but only work on single table.
